integration-tests/tart.py

```diff
@@ -0,0 +1,33 @@
+import tempfile
+import os
+import subprocess
+
+
+class Tart:
+    def __init__(self):
+        self.tmp_dir = tempfile.TemporaryDirectory()
```

Comment:
[`TemporaryDirectory()`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) doesn't seem to support that, unfortunately.

Perhaps we can just skip this for now, because macOS periodically cleans up temporary directory anyway?

Reply:
See https://github.com/cirruslabs/tart/pull/313/commits/1b7ae4a1d0048c299f2faa71cafd2dfcfb57cd14.